config/backends.go

```diff
@@ -49,23 +49,26 @@ const (
 )
 
 type Aerospike struct {
-	DefaultTTL int    `mapstructure:"default_ttl_seconds"`
-	Host       string `mapstructure:"host"`
-	Port       int    `mapstructure:"port"`
-	Namespace  string `mapstructure:"namespace"`
+	DefaultTTL int      `mapstructure:"default_ttl_seconds"`
+	Hosts      []string `mapstructure:"hosts"`
```

Comment:
Removal of `host` is a breaking change. Do you specifically require the use of multiple hosts? .. or is this perhaps just for completeness?

Reply:
Our use of aerospike is in a cluster. Specifying multiple hosts allow you to still connect even if one host is down.
I debated supporting both Host and Hosts, but for only having to change a string to an array in a config, I thought this would be a small enough breaking change to not acrue technical debt.

Thoughts?

Reply:
We try our best to avoid breaking changes that aren't absolutely necessary. I'm of the opinion we should leave the `host` config and add a new config, `additionalhosts` or some such. I think it's good to keep it an array under the covers, where the host config is prepended.

Reply:
ok, I can make that change.

Reply:
Thank you.

Reply:
An alternate, when you come back to this, is we change `host` to support a comma separate string.

But for now I added additional hosts field

Reply:
@SyntaxNode  Hi, What else can I do to help resolve this PR?